### DISCUSSION TITLE
NextJS App Router Course - Question where are the images stored?

### DISCUSSION THREAD

**Discussion by BearCooder** [vercel/next.js#59465] [Help] [answered]
### Summary

Hi! I am in the process of doing the Next JS course and right now at the point [Fetching data for LatestInvoices](https://nextjs.org/learn/dashboard-app/fetching-data#fetching-data-for-latestinvoices)

It works and I have no issues here. But however I am wondering where the` image_url: '/customers/lee-robinson.png',` are coming from? I see them in the placeholder-data.js but cannot find the actual images? Where are they stored? And how could they end up in the Vercel postgres database - I mean how can they be accessed as I do not see where they are? I actually feel stupid for asking that but I am somewhat confused about storing images in the database. I hope someone can help me please and clarify it?

**Comment by icyJoseph** [comment, marked as answer]
Hi,

- https://github.com/vercel/next-learn/tree/main/dashboard/starter-example/public/customers

**Reply by BearCooder** [reply]
I am so blind.. thank you!
